lib/rack/response.rb

```diff
@@ -19,22 +19,27 @@ module Rack
   # +write+ are synchronous with the Rack response.
   #
   # Your application's +call+ should end returning Response#finish.
-
   class Response
     def self.[](status, headers, body)
       self.new(body, status, headers)
     end
 
-    attr_accessor :length, :status, :body
-    attr_reader :header
-    alias headers header
-
     CHUNKED = 'chunked'
     STATUS_WITH_NO_ENTITY_BODY = Utils::STATUS_WITH_NO_ENTITY_BODY
 
-    def initialize(body = nil, status = 200, header = {})
+    # Initialize the response object with the specified body, status
+    # and headers.
+    #
+    # @param body [nil | #each | #to_str] the response body.
```

Comment:
As I see in https://rubydoc.info/gems/yard/file/README.md, there should be `,` instead of `|`, but I'm not sure.

Reply:
Fixed! Thanks.